examples/dynamics/shot_at_an_angle_to_the_horizon.py

```diff
@@ -0,0 +1,38 @@
+from sympy import solve, Symbol, Eq
```

Comment:
It is recommended to have this line at the beginning of each example: `#!/usr/bin/env python3`

Reply:
Added in 6951f71